config/webpack.dev.config.js

```diff
@@ -17,12 +16,6 @@ const presets = require('../lib/presets');
 const resolvePrivateEnvConfig = require('../lib/resolvePrivateEnvConfig');
 const getLocalAliases = require('./getLocalAliases');
 
-// Add process env vars. Currently used only for setting the
-// server port and the publicPath
-dotenv.config({
-  path: path.resolve(process.cwd(), '.env.development'),
-});
-
```

Comment:
You are right, I intended to remove those lines in another file because they were causing some issues during the build time, and by mistake, I removed them here as well. Thanks for noticing it.